satellite/orders/endpoint.go

```diff
@@ -46,6 +46,9 @@ type DB interface {
 	GetBucketBandwidth(ctx context.Context, projectID uuid.UUID, bucketName []byte, from, to time.Time) (int64, error)
 	// GetStorageNodeBandwidth gets total storage node bandwidth from period of time
 	GetStorageNodeBandwidth(ctx context.Context, nodeID storj.NodeID, from, to time.Time) (int64, error)
+
+	// ProcessOrders takes a list of order requests and processes them in a batch
+	ProcessOrders(ctx context.Context, requests []*ProcessOrderRequest) (responses []*pb.SettlementResponse, err error)
```

Comment:
Try to avoid protobuf types in database code.

Reply:
Creating a new PR for this